pypesto/objective/amici_objective.py

```diff
@@ -25,6 +25,7 @@ def __init__(self,
                  mapping_par_opt_to_par_sim=None,
                  mapping_scale_opt_to_scale_sim=None,
                  guess_steadystate=True,
+                 threads=1,
```

Comment:
```suggestion
                 n_threads: int = 1,
```
because otherwise in the class it will look like this holds a list of thread objects